<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1668702976722' stop='1668702979439'>
    <name>Data-Driven Test</name>
    <title>Data-Driven Test</title>
    <test-cases>
        <test-case start='1668702976722' status='broken' stop='1668702979438'>
            <name>should attempt to signup using data provided by faker</name>
            <title>should attempt to signup using data provided by faker</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='wdio'/>
                <label name='thread' value='0-0'/>
            </labels>
            <parameters>
                <parameter kind='argument' name='browser' value='chrome-107.0.5304.107'/>
            </parameters>
            <steps/>
            <attachments/>
            <failure>
                <message>faker.internet.confirmpassword is not a function</message>
                <stack-trace>TypeError: faker.internet.confirmpassword is not a function
                    at Context.&lt;anonymous> (C:\Users\colle\Desktop\colleen-brown-assignment4\test\specs\Assignment.js:54:186)
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>